scripts/tests/perf/setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euox pipefail
+
+# Establish a channel first. Helps the iperf3 test to be more stable.
```

Comment:
Nit: I think "channel" is not clear. Are you talking about TURN channels? In connlib, we call this a "connection". "tunnel" would also make sense (one tunnel has many connections, some of which may use TURN channels via one or more relays).

Reply:
Noted!